build_functions/CMakeLists.txt

```diff
@@ -266,3 +266,36 @@ function(add_reals_check_target)
         )
     endif()
 endfunction()
+
+# signature: install_library_with_prefix(<theTarget> <includePrefix> [ <public_header>... ]])
+# Creates an install target for library with a subdirectory and optional public header files.
+function(install_library_with_prefix theTarget includePrefix)
+    
+    if(ARGN)
+        set_target_properties(${theTarget} PROPERTIES PUBLIC_HEADER "${ARGN}")
+    endif()
+
+    # Set CMAKE_INSTALL_* if not defined
+    include(GNUInstallDirs)
+
+    if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+        set(CMAKE_INSTALL_LIBDIR "lib")
+    endif()
+
+    # Install elastic_log_rpc_client
+    set(package_location "cmake")
+
+    install(TARGETS ${theTarget} EXPORT ${theTarget}Targets
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR}/../bin
+        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${includePrefix}
+    )
+    #install(FILES ${zrpc_h_files} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/zrpc)
```

Comment:
>zrpc_h_files [](http://example.com/codeflow?start=21&length=12)

I get it this is in a comment but it is very specific to zrpc, so likely it should not be here at all